### PR TITLE
4703 – Remove ProfileLink

### PR DESCRIPTION
**Pull request by vasconsaurus**
## Description
Removes user profile link from item history.

### Context
While qa'ing Update @meedan people involvement in item history (PR here: https://github.com/meedan/check-api/pull/1972) we ran into a bug:
- When viewing item history and clicking on the user link, the ‘Meedan’ user link redirects to sometimes the ‘Meedan’ user and the current user.

After conversation in the ticket (4703), it was decided to get rid of the profile page. But, for the scope of that ticket it was decided to just remove the link to the profile page.

### Before and After

Screenshot 2024-08-06 at 11 10 54

Screenshot 2024-08-06 at 11 10 46

References: 4703

## Type of change

- [ ] Performance improvement and/or refactoring (non-breaking change that keeps existing functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Security mitigation or enhancement
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Automated test (add or update automated tests)

## How has this been tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can verify the changes. Please describe whether or not you implemented automated tests.

## Things to pay attention to during code review

Please describe parts of the change that require extra attention during code review, for example:

- File FFFF, line LL: This refactoring does this and this. Is it consistent with how it’s implemented elsewhere?
- Etc.

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] I've made sure my branch is runnable and given good testing steps in the PR description
- [ ] I considered secure coding practices when writing this code. Any security concerns are noted above.
- [ ] I have commented my code in hard-to-understand areas, if any
- [ ] I have made needed changes to the README
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I implemented any new components, they are self-contained, their `propTypes` are declared and they use React Hooks and, if data-fetching is required, they use Relay Modern with fragment containers
- [ ] If my components involve user interaction - specifically button, text fields, or other inputs - I have added a [BEM-like class name](https://meedan.atlassian.net/wiki/spaces/ENG/pages/1327628289/Naming+conventions+for+interactive+elements) to the element that is interacted with
- [ ] To the best of my knowledge, any new styles are applied according to the design system
- [ ] If I added a new external dependency, I included a rationale for doing so and an estimate of the change in bundle size (e.g., checked in https://bundlephobia.com/)

### CODE REVIEW

**Comment by vasconsaurus** [comment]
@amoedoamorim does this change make sense? Please let me know if you need more context, and if this doesn't make sense, needs to be changed in any other way. 🙏

File: src/app/components/annotations/Annotation.js [hunk 1]
```diff
@@ -77,7 +76,7 @@ class Annotation extends Component {
       ? <span className={cx('test-annotation__timestamp', styles['annotation-timestamp'])}><TimeBefore date={updatedAt} /></span>
       : null;
     const authorName = activity.user
-      ? <ProfileLink className="annotation__author-name" user={activity.user} /> : null;
+      ? <span className="annotation__author-name" user={activity.user}> {activity.user.name} </span> : null;
```
Comment (amoedoamorim) [inline src/app/components/annotations/Annotation.js:79]: The `user` prop only makes sense as passed to `ProfileLink` which expects a `user` prop. For the  `span` element you should pass the  `className` only.

**Review by amoedoamorim — CHANGES_REQUESTED** [review]
That's cool Manu! It's almost good, please see my comment about the user prop and fix that and it should be good to go!

File: src/app/components/annotations/Annotation.js (hunk 1 above)
Comment (vasconsaurus) [reply src/app/components/annotations/Annotation.js]: Makes sense! 😅

**Review by amoedoamorim — APPROVED** [review]
:lgtm:

Great to see you contributing to check-web!

**Review by danielevalverde — APPROVED** [review]

**Pull request merged**